.gitignore

```diff
@@ -33,6 +33,7 @@ npm-debug.log
 yarn-error.log
 testem.log
 /typings
+.env
```

Comment:
This is pretty good so far. I would recommend that you add a `.env.template` file to the project and push it to the this repo so that the database can be configured easily by any user.